code/game/machinery/cryopod.dm

```diff
@@ -708,6 +710,12 @@
 /obj/machinery/cryopod/blob_act()
 	return //Sorta gamey, but we don't really want these to be destroyed.
 
+/obj/machinery/cryopod/offstation
+	// Won't announce when used for cryoing.
+	silent = TRUE
+	orient_right = 1
+	icon_state = "body_scanner_0-r"
```

Comment:
Making these orient right purely for this ruin is meh. Make another instance for `offstation/right` or edit this on the DMM itself.

Reply:
Changed